core/src/main/java/org/elasticsearch/action/admin/cluster/node/stats/NodeStats.java

```diff
@@ -282,6 +295,12 @@ public void writeTo(StreamOutput out) throws IOException {
         out.writeOptionalStreamable(breaker);
         out.writeOptionalStreamable(scriptStats);
         out.writeOptionalStreamable(discoveryStats);
+        if (ingestStats == null) {
```

Comment:
Maybe I'm just not seeing it, but that one doesn't exist. There is only a variant for streamable.